docs/Onboarding/Prerequisites_Guide.md

```diff
@@ -0,0 +1,90 @@
+# (WIP) OpenShift CI Scenario Prerequisites Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Overview](#overview)
+- [Prerequisites at a Glance](#prerequisites-at-a-glance)
+- [Resources](#resources)
+- [Upstream Considerations](#upstream-considerations)
+  - [1. Public GitHub repositories](#1-public-github-repositories)
+  - [2. Private GitHub repositories](#2-private-github-repositories)
+- [GitHub Org Requirements](#github-org-requirements)
+- [GitHub User Readiness](#github-user-readiness)
+- [Containerized Product Deployment](#containerized-product-deployment)
+- [Containerized Tests](#containerized-tests)
+- [Resolve Internal Dependencies](#resolve-internal-dependencies)
+- [Valid Test Output](#valid-test-output)
+- [Documented Tests](#documented-tests)
+
+## Overview
+This doc is meant to be used by Product QE teams to prepare themselves and their testing repositories to run within OpenShift CI.
+
+You will be working through and updating the following information in the prerequisite Jira ticket that you will be assigned when starting this prerequisite process.
+
+>Please reach on the Slack channel [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) to notify us that you want to start the prerequisite process so that we can get you started with a prerequisite ticket.
+
+## Prerequisites at a Glance
+1. Resources
+2. Upstream Considerations
+3. GitHub Org Requirements
+4. GitHub User Readiness
+5. Containerized Product Deployment
+6. Containerized Tests
+7. Resolve Internal Dependencies
+8. Valid Test Output
+9. Documented Tests
+
+## Resources
+- Identify engineers that are assigned to work on & hold shared responsibility for product onboarding, test execution and ongoing scenario maintenance. See the [Scenario Maintenance Policy](../Policy/Maintenance/Scenario_Maintenance_Policy.md) and this [RACI Chart](RACI_Chart.md) to understand the roles & responsibilities for this onboarding.
+
+## Upstream Considerations
+Running in OpenShift CI is by default upstream and open to the public. All generated output and artifacts will be made public when run as a prow job. This means that the following MUST be true about your layered product deployment and test automation
+- No exposed secrets in the console output of any deployment, config, setup, or test automation.
+- No exposed secrets in an XML generated by a test.
+- No exposed secrets in any artifact generated from product deployment or product testing.
+
+There are two paths for onboarding your layered product scenario into OpenShift CI that you can take
+### 1. Public GitHub repositories
```

Comment:
I agree with putting preferred here it will at least get teams thinking about upstream first testing if they don't already do it. Also this model no need for mirroring making it less prone to failures, so I will make this change.